app/presenters/queries/content_item_presenter.rb

```diff
@@ -86,10 +100,11 @@ def extract_fields(scope)
           end
         end
 
-        scope = scope.select(*fields_to_select)
-        results = evaluate_query(scope)
+        scope.select(*fields_to_select)
+      end
 
-        parsing_enumerator(results)
+      def search(scope)
+        scope.where("title ilike ? OR base_path ilike ?", "%#{search_query}%", "%#{search_query}%")
       end
```

Comment:
Good catch 👍  my guard clause seems to have disappeared in the refactoring. Back now.